sdk/java/src/main/java/com/gojek/feast/FeastClient.java

```diff
@@ -125,7 +143,18 @@ public GetFeastServingInfoResponse getFeastServingInfo() {
         .map(
             field -> {
               Row row = Row.create();
-              field.getFieldsMap().forEach(row::set);
+              field
+                  .getFieldsMap()
+                  .forEach(
+                      (String name, Object value) -> {
```

Comment:
Are you intentionally using `Object` here or is there a more specific type we can use?

Reply:
We can use `ValueProto.Value` here. I used `Object` as `Row.set(String, Object)` used `Object`